pkg/inspector/image-inspector.go

```diff
@@ -298,3 +322,23 @@ func getAuthConfigs(dockercfg, username, password_file string) (*docker.AuthConf
 
 	return imagePullAuths, nil
 }
+
+func (i *defaultImageInspector) runOpenSCAP() []byte {
+	log.Printf("OpenSCAP scanning %s", i.opts.DstPath)
+	scanner := openscap.NewDefaultScanner(i.opts.DstPath, OSCAP_CVE_DIR,
+		i.opts.OscapResultsArf, &i.meta.Image)
+	err := scanner.Scan()
+	if err != nil {
+		fmt.Printf("Unable to run OpenSCAP: %v\n", err)
```

Comment:
Please return the error and allow the caller to decide how to handle them (same for below)